src/lineshape.h

```diff
@@ -91,28 +95,33 @@ class Lorentz {
   }
 }; // Lorentz
 
-class Voigt {
+struct Voigt {
   Numeric mF0;
   Numeric invGD;
   Complex z;
 
-public:
   Complex F;
 
-private:
   Complex dF;
 
-public:
-  Voigt(Numeric F0_noshift, const Output &ls, Numeric DC, Numeric dZ) noexcept;
+  constexpr Voigt(Numeric F0_noshift, const Output &ls, Numeric DC, Numeric dZ) noexcept
+  : mF0(F0_noshift + dZ + ls.D0 + ls.DV),
+  invGD(1.0 / std::abs(DC * mF0)), z(invGD * Complex(-mF0, ls.G0)), F(), dF() {}
```

Comment:
The needs to use `nonstd::abs` as well to compile with libc++. Really annoying that they still haven't made these math functions `constexpr` in the standard. To spot compatibility issues with `libstc++`, I'll make a PR to adapt our clang test build to use `libc++` instead of `libstdc++`.

Reply:
Fixed now.